### PR TITLE
More UX fine tuning

### PR DESCRIPTION
**Pull request by mounikasomisetty**
https://screencast.googleplex.com/cast/NjY0NzcwODQyNzIyMzA0MHxhZGRkMGNhNC03Zg
+ Fixed spacing in side bar header and view result button
+ dividers full length fix
+ week scheduler spacing fix
+ bump shared version

### CODE REVIEW

File: jupyterlab_gcpscheduler/package.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
     "@phosphor/signaling": "^1.3.0",
     "csstips": "^1.2.0",
     "formik": "^1.5.8",
-    "gcp_jupyterlab_shared": "1.1.2",
+    "gcp_jupyterlab_shared": ">=1.1.2",
```
Comment (prodonjs) [inline jupyterlab_gcpscheduler/package.json:55]: I think using caret `^1.1.2` is best here since it will ensure we remain on the same major version.

**Review by prodonjs — CHANGES_REQUESTED** [review]
Just a small suggestion in the `package.json` file. Also, let's switch the Badge to use _Experimental_ instead of Alpha since that's the nomenclature we are using these days.

File: jupyterlab_gcpscheduler/package.json (hunk 1 above)
Comment (mounikasomisetty) [reply jupyterlab_gcpscheduler/package.json]: Ah okay. I thought since 1.1.2 had a breaking change (more icons) it had to be >=. Changed!

**Comment by mounikasomisetty** [comment]
http://screen/3rcEEKDtQ5mxocN
http://screen/ANXc9yXJbrSMkKE
Changed 'alpha' to experimental

**Review by prodonjs — CHANGES_REQUESTED** [review]
LGTM but I think you may need to update some snapshots after the badge change.

**Review by prodonjs — APPROVED** [review]

**Pull request merged**